src/components/session/companies.js

```diff
@@ -0,0 +1,63 @@
+import React from "react"
+import Img from "gatsby-image"
+import { graphql } from "gatsby"
+import { Container, Row, Col } from "reactstrap"
+
+import CompaniesStyles from "../../styles/session/companies.module.css"
+import { useStaticQuery } from "gatsby"
+
+// const getCompaniesArrayString = companies => {
```

Comment:
Is this code needed for later?

Reply:
No, I don't need it anymore. It's now deleted! Thanks!